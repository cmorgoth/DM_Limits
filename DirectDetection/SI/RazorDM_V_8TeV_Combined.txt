weight 1.0
1	948.719
10	943.234
100	939.029
400	800.769
700	614.977
1000	430.361
