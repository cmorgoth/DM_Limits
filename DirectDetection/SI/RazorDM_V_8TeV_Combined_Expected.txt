weight 1.0
1	1117
10	1117.59
100	1111.53
400	972.607
700	755.968
1000	535.997
